src/ontology/edits/oeo-shared.omn

```diff
@@ -2435,6 +2435,26 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1463",
         OEO_00000149
     
     
+Class: OEO_00000407
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A technology is a plan specification that describes how to combine artificial objects or other material entities and processes in a specific way."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/136
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/334
+moved to oeo-shared, change definition, reclassify, add axiom
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1572
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1591",
+        rdfs:label "technology"
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/IAO_0000104>,
+        <http://purl.obolibrary.org/obo/IAO_0000136> some 
+            (OEO_00000061
+             and (<http://purl.obolibrary.org/obo/RO_0000056> some 
+                (OEO_00000419 or OEO_00140003)))
```

Comment:
According to the definition of `technology` (_...to combine artificial objects or other material entities and **processes** in a specific way_) I would prefer to have here the axiom `'is about' some ('artificial object' and ('participates in' some process)`. The narrowing down to specific processes comes then in the specific technology subclasses.

Reply:
ok